public/images/mozilla.svg

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
It's just something weird with github's display, right click the image in the above link and click view image, should be fine.